modules/internal/String.chpl

```diff
@@ -233,6 +233,23 @@ module String {
     var _cpindex  : int;
   }
 
+  // Helper routines in support of being able to use ranges of codePointIndex
+  proc chpl_build_bounded_range(low: codePointIndex, high: codePointIndex)
+    return new range(codePointIndex, _low=low, _high=high);
+  proc chpl__rangeStrideType(type idxType: codePointIndex) type
+    return int;
+  proc chpl__rangeUnsignedType(type idxType: codePointIndex) type
+    return uint;
+  inline proc chpl__idxToInt(i: codePointIndex)
+    return i:int;
+  inline proc chpl__intToIdx(type idxType: codePointIndex, i: int)
+    return i: codePointIndex;
+  proc chpl__idxTypeToIntIdxType(type idxType: codePointIndex) type
+    return int;
+  pragma "no doc"
```

Comment:
I think all of these above need `no doc` too. They're currently rendered in the documentation.

Reply:
Huh... you're right.  I'd really gotten it into my mind that chpldoc skipped over `chpl_` prefixed things unless you were in developer mode (and thought I'd seen evidence of that in some other edits I'd done in other files...).

Reply:
The horrid [fixInternalDocs.sh](https://github.com/chapel-lang/chapel/blob/master/modules/internal/fixInternalDocs.sh) has a `removePrefixFunctions` routine that removes all chpl_ prefixed routines and any operators. It's not used on the string module because we want to document the operator overloads.

Reply:
OK, glad I didn't completely fabricate that impression!

Reply:
nodoc'd in 0238a805b243a1408e5bccb7cc9f458f665429d8